lib/api/html/accessibility.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * This file contains Beans accessibility features.
+ *
+ * @package Beans\Framework\API\HTML
+ *
+ * @since   1.5.0
+ */
+
+/**
+ * Build the skip links.
+ *
+ * @since 1.5.0
+ *
+ * @return void
+ */
+function beans_build_skip_links() {
+	$skip_links = array();
+	$layout     = beans_get_layout();
+
+	if ( has_nav_menu( 'primary' ) ) {
+		$skip_links['beans-primary-navigation'] = __( 'Skip to the primary navigation.', 'tm-beans' );
+	}
+
+	$skip_links['beans-content'] = __( 'Skip to the content.', 'tm-beans' );
+
+	if ( 'c' !== $layout ) {
+
+		if ( in_array( $layout, array( 'c_sp', 'sp_c', 'c_sp_ss', 'sp_c_ss', 'sp_ss_c' ), true ) && beans_is_active_widget_area( 'sidebar_primary' ) ) {
```

Comment:
Line 29 knows way too much about the Layout API.  

Instead, let's move this code to the Layout API `functions.php` like this:

```
/**
 * Check if the given layout has a primary sidebar.
 *
 * @since 1.5.0
 *
 * @param string $layout The layout to check.
 *
 * @return bool
 */
function beans_has_primary_sidebar( $layout ) {

	if ( ! in_array( $layout, array( 'c_sp', 'sp_c', 'c_sp_ss', 'sp_c_ss', 'sp_ss_c' ), true ) ) {
		return false;
	}

	return beans_is_active_widget_area( 'sidebar_primary' );
}
```

Then here on Line 29, you can use that new function:

```
if ( beans_has_primary_sidebar( $layout ) ) {
```